.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy to Kubernetes
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Kubernetes
+      uses: azure/setup-kubectl@v3
+      with:
+        version: 'latest'
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: 'latest'
+
+    - name: Create .kube directory
+      run: mkdir -p $HOME/.kube
+
+    - name: Authenticate with Kubernetes
+      env:
+        KUBECONFIG: ${{ secrets.KUBECONFIG }}
+      run: echo "${{ secrets.KUBECONFIG }}" > $HOME/.kube/config
+
+    - name: Package Helm chart
+      run: helm package ./codbex-athena
+
+    - name: Push Helm chart to GitHub Container Registry
+      env:
+        CR_PAT: ${{ secrets.GH_TOKEN }}
+      run: |
+        export CHART_VERSION=$(cat ./codbex-athena/Chart.yaml | grep version | awk '{print $2}')
+        export IMAGE_NAME="ghcr.io/${{ github.repository }}/codbex-athena-chart"
+        helm push ./codbex-athena-${CHART_VERSION}.tgz oci://${IMAGE_NAME}
+
+    - name: Deploy Helm chart
+      env:
+        CR_PAT: ${{ secrets.GH_TOKEN }}
+      run: |
+        helm registry login ghcr.io -u ${{ github.actor }} -p ${{ secrets.GH_TOKEN }}
+        export IMAGE_NAME="ghcr.io/${{ github.repository }}/codbex-athena-chart"
+        helm install codbex-athena oci://${IMAGE_NAME} --namespace dev --create-namespace
```

Comment:
Please stick to the other way of deploying containers to the cluster, let's not overcomplicate things by adding Helm charts :)